src/ai/script_ai.cpp

```diff
@@ -598,11 +622,21 @@ static int CclAiNeed(lua_State *l)
 }
 
 /**
+** <b>Description</b>
+**
 **  Set the number of units.
 **
 **  @param l  Lua state
 **
 **  @return   Number of return values
+**
+** Example:
+**
+** <div class="example">
+** 	<code>
+**		peasants = AiSet("unit-peasant",3)
```

Comment:
Same here, I think the code example could lead to confusion.

Reply:
Maybe this is better?
`function() return AiSet(AiWorker(), 2) end`